pkg/controller/volume/expand/expand_controller.go

```diff
@@ -216,6 +216,13 @@ func (expc *expandController) pvcUpdate(oldObj, newObj interface{}) {
 			eventType := v1.EventTypeNormal
 			if err != nil {
 				eventType = v1.EventTypeWarning
+			} else {
+				// Do not log and record the event when we can not found the plugin
+				// and the StorageClassName is not set at the same time.
+				// See https://github.com/kubernetes/kubernetes/issues/77404
+				if len(pv.Spec.StorageClassName) == 0 {
```

Comment:
I think that on this place recording event `didn't find a plugin capable of expanding the volume` is appropriate. Someone has changed `pvc.spec.Resources` and expects resize. We could tweak the message to be more clear.